internal/backoff/backoff.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) F5, Inc.
+//
+// This source code is licensed under the Apache License, Version 2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+
+package backoff
+
+import (
+	"context"
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
+)
+
+const (
+	Jitter     = 0.10
+	Multiplier = backoff.DefaultMultiplier
+)
+
+type Backoff[T any] struct{}
+
+type Settings struct {
```

Comment:
Would it be better to reuse the CommonSettings struct from types.go instead of creating this Settings struct?

Reply:
I kept these separate so people could reuse them easily if we externalize the WaitUntil function at some point. I think we might. It will be useful in the Management Server and you don't want to depend on an Agent Config for that